### ISSUE TITLE
Make version 0.9.3 available for download

### ISSUE THREAD

**Issue by Lineflyer** [cgeo/WhereYouGo#87]
We should make this old version available to users via some download location to allow fallback in case they are affected by the regressions described in other issues.

Need to check tomorrow If I have the APK of 0.9.3. somwhere or if its stored on the old repo.

**Comment by jakedot** [comment]
See https://github.com/biylda/WhereYouGo/releases/tag/0.9.3-beta

**Comment by allyourcodearebelongtous** [comment]
Downloaded a working 0.9.3 beta via https://whereyougo.de.aptoide.com/ (if you dare install APKs from unknown sources) and downgraded the failing WhereYouGo Player while installing the older beta.
This should of course only be neccessary for Android Oreo (and maybe older versions).

**Comment by jakedot** [comment]
Is there a problem with using the official release I linked to above?

**Comment by Lineflyer** [comment]
Thanks for pointing me to it. I included this into our repository release history.

In fact I had to deleted and recreate all releases and tags to enable the correct sequence as despite using the matching commit (from 2017) github always uses the tag creation date/time instead of the commit creation date/time.

But anyhow the old release is now here:
https://github.com/cgeo/WhereYouGo/releases/download/0.9.3-beta/whereyougo-0.9.3-signed.apk
